packages/components/pie-notification/test/component/pie-notification.spec.ts

```diff
@@ -235,6 +249,237 @@ test.describe('PieNotification - Component tests', () => {
                 });
             });
         });
+
```

Comment:
These tests should probably be visual tests rather than component tests. If you need a hand setting these up, please reach out!

Reply:
@siggerzz I just created visual tests adding based on the unit tests. I think maintaining both visual and unit tests enriches the test coverage.